image_transport/src/publisher.cpp

```diff
@@ -110,19 +110,36 @@ Publisher::Publisher(
   impl_->base_topic_ = image_topic;
   impl_->loader_ = loader;
 
-  std::vector<std::string> blacklist_vec;
-  std::set<std::string> blacklist;
-  // nh.getParam(impl_->base_topic_ + "/disable_pub_plugins", blacklist_vec);
-  for (size_t i = 0; i < blacklist_vec.size(); ++i) {
-    blacklist.insert(blacklist_vec[i]);
+  uint ns_len = node->get_effective_namespace().length();
+  std::string param_base_name = image_topic.substr(ns_len);
+  std::replace(param_base_name.begin(), param_base_name.end(), '/', '.');
+  if (param_base_name.front() == '.') {
+    param_base_name = param_base_name.substr(1);
   }
-
+  std::vector<std::string> whitelist_vec;
```

Comment:
Please use the term "allowlist" instead of "whitelist".

Reply:
@gbiggs I addressed [here](https://github.com/ros-perception/image_common/pull/264/commits/f24fa62231bb5623a6639560a8fd388de130e580)